.github/workflows/build_releasecandidate.yml

```diff
@@ -54,6 +54,9 @@ jobs:
         rm -rf ./build/tmp
         cp ./docs-ext/curriculum-*.pdf ./build 2>/dev/null || :
         zip -r release.zip ./build
+        mkdir release_dir
+        cp ./build/curriculum-*.pdf ./release_dir 2>/dev/null || :
+        mv release.zip ./release_dir/release-${{ env.RELEASE_VERSION }}.zip
```

Comment:
hm, I remember having had issues with underscores in Github-Pages URIs. Let's keep our fingers crosse that this works, otherwise replace with hyphen

Reply:
Yap, it's not part of GitHub Pages or anything, just a storage for the release files. It works, look at the latest RC. 🙂